content/blogs/2023-10-05-announcing-Kestra-funding-to-build-the-universal-open-source-orchestrator.md

```diff
@@ -0,0 +1,118 @@
+
+Software engineers appreciate the **API-first** design and **robust**, **scalable**, **cloud-native** architecture of our product. **Event-driven** capabilities are within Kestra’s DNA, enabling the creation, execution, and real-time monitoring of workflows at any scale. Kestra can efficiently process millions of events and ensures that businesses can reliably expand without sacrificing performance.
+
+## Run Anywhere, Code in Any Language
+
+**Les languages des programmations Kestra en images + cloud/premise**
+
+In terms of language compatibility, Kestra supports tasks written in a wide array of programming languages, including **Python**, **R**, **Node.js**, **Julia**, **Rust**, **Bash**, **PowerShell**, and anything that can run as a containerized process. Kestra's flows can run anywhere — from your laptop or on-prem VM to a Kubernetes cluster in a public cloud. Thanks to the extensible integrations ecosystem, including features such as custom plugins, triggers, tasks, blueprints, and fully customizable scripts and microservices, the product keeps evolving along with the business requirements of our users.
+
+## Infrastructure as Code Meets Orchestration
+
+**CI/CD image**
+
+In contrast to orchestrators that confine workflow creation to a single programming language, Kestra leverages a **declarative YAML interface** that democratizes orchestration and facilitates participation from developers utilizing diverse technology stacks. What our users appreciate most is that our product brings Infrastructure as Code best practices to all workflows — from those orchestrating mission-critical operations, business processes, and data pipelines to simple Zapier-style automations. Everything can be version-controlled and governed using software engineering best practices. Whether you make changes through the user interface, **CI/CD**, or via API calls, those changes are always automatically reflected in the codebase. This API-first approach fosters a synergistic workspace, allowing our users to build an internal platform loved by engineers and business stakeholders.
+
+## Fast, Reliable, and Extensible
+
+Kestra strives to be the easiest-to-use orchestration tool on the market. Our users construct their first scheduled or event-driven workflow in a matter of minutes via our intuitive user interface or API, thereby obviating the need for complex deployment processes. We are steadfast in our belief that a simple orchestration API leads to greater reliability and extensibility than convoluted client-side frameworks.
+
+## What's Coming Next
+
+The introduction of a SaaS offering is among our upcoming milestones, further simplifying the user experience by removing initial setup complexities. This will serve as an additional option alongside our existing solutions, catering to a wider range of user needs.
```

Comment:
```suggestion
The introduction of a fully-managed offering is among our upcoming milestones, further simplifying the user experience by removing the complexity of managing infrastructure. This will serve as an additional option alongside our existing [Kestra Enterprise](url) solution.
```